pkgs/applications/science/machine-learning/deeplabcut/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, wrapGAppsHook
+, opencv3
+, python3
+, mkl
+ }:
+
+let
+  opencv3Ffmpeg = opencv3.override {enableFfmpeg = true;};
+        
+  pythonOverrides = python-self: python-super: {
+    # openblas does not work https://github.com/AlexEMG/DeepLabCut/issues/637#issuecomment-606893058
+    numpy = python-super.numpy.override { blas = mkl; };
+    # imgaug test stalls...
+    imgaug = python-super.imgaug.overrideAttrs (old:{
+      doInstallCheck = false;
```

Comment:
Should this be pushed up to the package derinition?

Reply:
@veprbl the hang may be specific to MKL (not positive), so I took the more conservative approach..